src/sentry/auth_v2/urls.py

```diff
@@ -31,4 +32,9 @@
         AuthUserMergeVerificationCodeEndpoint.as_view(),
         name="sentry-api-0-auth-verification-codes",
     ),
+    re_path(
+        r"^merge-accounts/$",
```

Comment:
Nit, but you don't have to change anything since these are short-term endpoints.

Think of the URL path as a file structure. Which is more obvious that the endpoints that serve a common user-flow?
- `api/.../merge/verification-code` + `api/.../merge/`,  
- `api/.../user-merge-verification-code/` + `api/.../merge-accounts/`